src/app/layers/qgsapplayerhandling.cpp

```diff
@@ -175,7 +175,7 @@ void QgsAppLayerHandling::addSortedLayersToLegend( QList<QgsMapLayer *> &layers
     } );
   }
 
-  QgsLayerTreeGroup *parent = nullptr;
+  /*QgsLayerTreeGroup *parent = nullptr;
   QgsLayerTreeNode *currentNode = QgisApp::instance()->layerTreeView()->currentNode();
```

Comment:
Is this a wip?

Reply:
Nope, it's a push that was missing --force and failed without me noticing

Reply:
@nyalldawson , fixed